### ISSUE TITLE
Error when resetting a database (disappears after second call)

### ISSUE THREAD

**Issue by epogrebnyak** [epogrebnyak/cbr-db#19]
```
d:\git\cbr-db\py3>python bankform.py reset database
Database: dbf_db3
Traceback (most recent call last):
  File "bankform.py", line 224, in <module>
    main(sys.argv[1:])
  File "bankform.py", line 135, in main
    general_database_operations(arg, db_name)
  File "bankform.py", line 125, in general_database_operations
    delete_and_create_db(db_name)
  File "d:\git\cbr-db\py3\database.py", line 141, in delete_and_create_db
    execute_sql(command)
  File "d:\git\cbr-db\py3\conn.py", line 40, in execute_sql
    resp = execute_sql_with_cursor(sql_string, cur, verbose)
  File "d:\git\cbr-db\py3\conn.py", line 26, in execute_sql_with_cursor
    cur.execute(sql_string)
  File "D:\Anaconda3\lib\site-packages\pymysql\cursors.py", line 134, in execute
    result = self._query(query)
  File "D:\Anaconda3\lib\site-packages\pymysql\cursors.py", line 283, in _query
    self._do_get_result()
  File "D:\Anaconda3\lib\site-packages\pymysql\cursors.py", line 298, in _do_get_result
    self._show_warnings(conn)
  File "D:\Anaconda3\lib\site-packages\pymysql\cursors.py", line 302, in _show_warnings
    for w in ws:
TypeError: 'NoneType' object is not iterable
```

**Comment by repo2156** [comment]
I can't reproduce this issue here. It occurs at every first execution (after starting MySQL server) of database reset?

**Comment by epogrebnyak** [comment]
It occurs when database is empty - no dbf_db3 and cbr_db3 databases are present. On the first run ```python bankform.py reset database``` there is this error, on  second run there is no error.

**Comment by repo2156** [comment]
I've reproduced it here, and fixed it in the config-folders branch (did not pushed yet to master as this is not critical and can wait a future merge).

It seems to be an odd behavior with pymysql, as even using the "IF EXISTS" syntax, it still raises a warning. I've separated the previous command in two, so the warning of the first does not triggers that bug.

Fixed (but the 'harmless' warning is still printed).
